src/feature/dirauth/dirvote.c

```diff
@@ -4620,7 +4620,8 @@ dirserv_generate_networkstatus_vote_obj(crypto_pk_t *private_key,
 
   v3_out->known_flags = smartlist_new();
   smartlist_split_string(v3_out->known_flags,
-                         "Authority Exit Fast Guard Stable V2Dir Valid HSDir",
+                         "Authority Exit Fast Guard Stable V2Dir Valid HSDir "
+                         "StaleDesc",
```

Comment:
Should this string be in alphabetical order for readability?
Does it matter?

Reply:
It doesn't matter, since we sort the list later, but I agree that it helps clarity.  I'll add a commit to alphabetize it after my plane lands.